src/pages/match.tsx

```diff
@@ -69,17 +70,17 @@ const MatchInternal: FC<Props> = ({ roomId, session }) => {
     await vote.mutateAsync({ selectedUserIds, roomId });
   };
 
-  const splashVariant = room.stage === "results" ? undefined : room.stage;
+  const splashVariant = room.stage === RESULTS ? undefined : room.stage;
 
   return (
-    <Layout splashScreenVariant={splashVariant}>
+    <Layout splashScreenVariant={splashVariant} localPlayer={localPlayer}>
       <PlayersOthers
         room={room}
         localPlayer={localPlayer}
         isVoteEnabled={isVoteEnabled}
         onVote={handleVote}
       />
-      {room.stage === "results" && (
+      {room.stage === RESULTS && (
```

Comment:
This change is useless and only adds confusion on the long run. The stage is a union of literals (not a string) thus `"results"` isn't just a magic string, but one of the possible values allowed and it is statically recognised by the parser